zkevm/src/circuit.rs

```diff
@@ -242,3 +249,74 @@ impl TargetCircuit for PoseidonCircuit {
         hash_rows
     }
 }
+
+pub struct ByteCodeCircuit {}
+impl TargetCircuit for ByteCodeCircuit {
+    type Inner = ByteCodeCircuitImpl<Fr>;
+
+    fn name() -> String {
+        "ByteCode".to_string()
+    }
+
+    fn empty() -> Self::Inner {
+        let r = tests::get_randomness();
```

Comment:
so the `r` here will be different from block.randomness.  Even it works now, i think let them be same like would be better. Ref: https://github.com/scroll-tech/common-rs/blob/216e372602175f17d8cdbb2e12f443ff268e0c24/zkevm/src/circuit.rs#L118

Reply:
`tests::get_randomness() ` return the `DEFAULT_RAND`  so it pass

Reply:
update to use `DEFAULT_RAND` directly in [c93e757](https://github.com/scroll-tech/common-rs/pull/61/commits/c93e757d6c57a85dac5a45a693651a5f4d11809c)